test/functional/apps/management/exports/_import_objects_with_index_pattern_conflicts.json

```diff
@@ -0,0 +1,19 @@
+[
```

Comment:
Where is this used?

Reply:
This was checked in by mistake. I will remove.